config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -255,7 +255,7 @@ periodics:
     testgrid-tab-name: cos-cgroupv1-containerd-node-e2e-serial
 - name: ci-containerd-node-e2e-1-5
```

Comment:
we will remove this job soon as containerd 1.5 is EOL. Changeing interval is ok. Thank you for fixing the flags